diff --git a/config/server-runtime.jsx b/config/server-runtime.jsx
index a06e7cb9cb549aa66f122eb160eb0dd48cb647fb..cf24addbcc28a72c78b8eb9d54f7e3d1db86b853 100644
--- a/config/server-runtime.jsx
+++ b/config/server-runtime.jsx
@@ -55,7 +55,7 @@ let INSTANCE = 0;
 
 function createRequest(base, id, instance, body, contentType) {
   return fetch(base, {
-    method: "POST",
+    method: "GET",
     headers: {
       "x-server-id": id,
       "x-server-instance": instance,
